packages/button/README.md

```diff
@@ -1,44 +1,41 @@
 # Buttons
 
-Buttons are used for action items. Some examples include Create, Save, Delete. Buttons make it obvious to the user that they can take an action.
+Buttons are essential to enable users perform actions. Buttons should be used for action items such as Create, Save, Delete, and should not be used for navigational elements. Instead, use links to take the user to a new page.
 
-Do not use buttons as navigational elements. Instead, use anchors because it takes the user to a new page and is not associated with an action.
+## Types of Buttons
 
-Each page will likely have one primary button. Any remaining calls-to-action are represented as secondary buttons.
+There are several types of buttons to choose from based on the intended action:
 
-## Button states and types
+| Type            | Purpose                                                                         |
+| --------------- | ------------------------------------------------------------------------------- |
+| Primary         | Used for the most important call-to-action on a page, such as Save or Deploy.   |
+| Secondary       | Most buttons on a page are likely to be this style.                             |
```

Comment:
Is it worth giving a bit more info on when to use the Secondary button? Maybe something like, `Used for offering non-primary actions to users, such as Cancel, View Details, etc.`